include/pingcap/kv/Cluster.h

```diff
@@ -24,6 +24,8 @@ struct Cluster
 
     LockResolverPtr lock_resolver;
 
+    std::unordered_set<uint64_t> min_commit_ts_pushed;
```

Comment:
Is it thread-safe ?

Reply:
**Cluster** is shared by every caller for this cluster. It's better to put it in `ResponseIter` ?

Reply:
Fixed.